README.md

```diff
@@ -10,5 +10,5 @@ For more information on Blackbaud's SKY Add-in framework, see https://developer.
 - Go through the <a href="https://developer.blackbaud.com/skyux/learn/get-started" target="_blank">SKY UX Getting Started</a> guide
```

Comment:
Go ahead and remove `This code is running live at https://host.nxt.blackbaud.com/addin-entry-form-demo.` above